source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -330,6 +337,17 @@ export default class WalletSendForm extends Component<Props, State> {
               this.setReceiverValidity(isValid);
               const adaAmountField = form.$('adaAmount');
               const isAdaAmountValid = adaAmountField.isValid;
+              const { selectedAsset } = this.props;
+              const shouldFocus = !this.isAddressFromSameWallet();
+              if (selectedAsset && !shouldFocus) {
+                const newAsset = `asset_${selectedAsset.fingerprint}`;
+                const newAssetField = this.form.$(newAsset);
+                const { name: fieldName } = newAssetField;
+                const focusableField = this.focusableFields[fieldName];
+                if (focusableField && focusableField.inputElement) {
+                  focusableField.inputElement.current.blur();
```

Comment:
@aleksandardjordjeviciohk this feel quite hacky...

Reply:
Updated